.eslintrc.js

```diff
@@ -42,6 +42,13 @@ module.exports = {
 		'@wordpress/dependency-group': 'error',
 		'@wordpress/gutenberg-phase': 'error',
 		'@wordpress/react-no-unsafe-timeout': 'error',
+		'@wordpress/i18n-text-domain': [
+			'error',
+			{
+				allowedTextDomains: [ 'default' ],
```

Comment:
Would it be fair to say that the majority usage would be to provide a single domain value for this option?

Wondering if we should optimize for this by accepting a single value, naming the option in the singular form, and optionally accepting an array as an overloaded form:

```
allowedTextDomain: 'default'
// Equivalent to:
// allowedTextDomain: [ 'default' ]
```

From the implementation, we can still normalize this to an array regardless of what's given:

```
const allowedTextDomains = lodash.castArray( options.allowedTextDomain );
```

Reply:
> Would it be fair to say that the majority usage would be to provide a single domain value for this option?

Definitely a fair assumption.

> Wondering if we should optimize for this by accepting a single value, naming the option in the singular form, and optionally accepting an array as an overloaded form

JSON schema seems to support that. I'll give it a shot 👍

Reply:
Done in e7187a0071013a35b2afaa9ecae72a9579eb9625